Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -150,24 +151,45 @@ public BoundingBoxActivationType BoundingBoxActivation
             }
         }
 
-        [SerializeField]
-        [Tooltip("Maximum scaling allowed relative to the initial size")]
-        private float scaleMaximum = 2.0f;
         [SerializeField]
         [Tooltip("Minimum scaling allowed relative to the initial size")]
         private float scaleMinimum = 0.2f;
+        [SerializeField]
+        [Tooltip("Maximum scaling allowed relative to the initial size")]
+        private float scaleMaximum = 2.0f;
+
 
         /// <summary>
-        /// Public property for the scale maximum, in the target's local scale.
+        /// Public property for the scale minimum, in the target's local scale.
         /// Set this value with SetScaleLimits.
         /// </summary>
-        public float ScaleMaximum => maximumScale.x;
```

Comment:
>public float ScaleMaximum => maximumScale.x; [](start = 8, length = 44)

This change also makes it so it's no longer possible to set the scale min/maxes after the fact (is that the intention?)

Reply:
Setting the min/max values after the fact is still possible using `SetScaleLimits`.